oecophylla/taxonomy/parser.py

```diff
@@ -1,7 +1,26 @@
+import sys
```

Comment:
do we really need "sys"?

Reply:
Well you did it...

Reply:
can you remove and run the tests. I doubt that we need to have it here.

Reply:
Okay did.